modin/core/io/text/text_file_dispatcher.py

```diff
@@ -1101,7 +1115,6 @@ def _read(cls, filepath_or_buffer, **kwargs):
         partition_kwargs = dict(
             kwargs,
             fname=filepath_or_buffer_md,
-            num_splits=num_splits,
```

Comment:
Why removed?

Reply:
This parameter is now explicitly passed to `_launch_tasks` function. This had to be done because `partition_kwargs` is now passed to `_launch_tasks` function as a reference/future.